pkg/ARCHITECTURE.md

```diff
@@ -31,6 +31,10 @@ We value clean & readable code that is loosely coupled and covered by unit tests
 The majority of our tests uses go convey but thats something we want to avoid going forward.
 For new tests we want to use standard library and `testify/assert`.
 
+## Import aliases
+Currently there are import aliases for `/pkg/models` package but thats something we want to avoid going forward.
+Newly introduced code should refer explicitly to the `models` instead of using the alias `m`.
```

Comment:
Can you add something in new sentence at the end along the lines: 

```suggestion
Newly introduced code should refer explicitly to the `models` instead of using the alias `m`. Whenever changing existing code it's desired to remove the import aliases as well.
```

What do you think?